tools/js-sdk-release-tools/src/common/utils.ts

```diff
@@ -66,20 +66,30 @@ function printErrorDetails(
     }
 }
 
+function getNpmClassicClientParametersPath(packageRoot: string): string {
+    return path.join(packageRoot, 'dist/esm/models/parameters.js');
+}
+
 export const runCommandOptions: SpawnOptions = { shell: true, stdio: ['pipe', 'pipe', 'pipe'] };
 
 export function getClassicClientParametersPath(packageRoot: string): string {
     return path.join(packageRoot, 'src', 'models', 'parameters.ts');
 }
 
+// NOTE: due to migration tool, the folder structure is changed, and src folder is removed
 export function getSDKType(packageRoot: string): SDKType {
-    const paraPath = getClassicClientParametersPath(packageRoot);
     const packageName = getNpmPackageName(packageRoot);
     if (packageName.startsWith('@azure-rest/')) {
         return SDKType.RestLevelClient;
     }
-    const exist = shell.test('-e', paraPath);
-    const type = exist ? SDKType.HighLevelClient : SDKType.ModularClient;
+    
+    const paraPath = getClassicClientParametersPath(packageRoot);
+    const npmParaPath = getNpmClassicClientParametersPath(packageRoot);
```

Comment:
would it be problematic if the latest npm version is HLC after the esm migration and we are moving to modular with TypeSpec in the current PR?

Reply:
Oh, forget to rename the function, the current behavior is to check both `src/models/parameters.ts` and `dist/esm/models/parameters,js`, if one of them exists, the sdk is HLC.